src/utils/hooks/useKubevirtAlerts.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import { useDashboardResources } from '@openshift-console/dynamic-plugin-sdk-internal';
+import { Alert } from '@openshift-console/dynamic-plugin-sdk-internal/lib/api/common-types';
+
+import { OPERATOR_LABEL_KEY } from '../../views/clusteroverview/OverviewTab/status-card/utils/constants';
+
+const isKubeVirtAlert = (alert) => alert?.labels?.[OPERATOR_LABEL_KEY] === 'kubevirt';
+
+const inNamespace = (namespace, alert) =>
+  namespace ? alert?.labels?.namespace === namespace : alert;
```

Comment:
please check if only alert?.labels?.namespace === namespace can wrok.. :)

Reply:
It wouldn't. This function needs to return true if namespace isn't specified. I refactored the hook a little bit to make the desired functionality clearer. It also has the side effect of making this function work the way you wrote it here. =)